test/integration/targets/package_facts/tasks/main.yml

```diff
@@ -18,6 +20,27 @@
     - name: check for ansible_facts.packages exists
       assert:
         that: ansible_facts.packages is defined
+
+    - name: Now try again but installing misleading rpm
+      block:
+        - name: install misleading rpm api
+          package: name="python3-rpm" state=present
+
+        - name: install misleading rpm 'binary' file
+          file: dest="{{output_dir}}/rpm" state=touch mode='0700'
```

Comment:
needs mkdir first?
```console
Error, could not touch target: [Errno 2] No such file or directory: b'/root/ansible/test/results/.tmp/output_dir/rpm'
```

Reply:
no, i'm probably missing setup, OUTPUT_DIR is a stadnard of our tests

Reply:
forgot, output_dir is only for controller